docs/integrations/community-nodes/risks.md

```diff
@@ -0,0 +1,17 @@
+# Risks when using community nodes
+
+Using community nodes has some risks. You are installing unverified code from a public source into your n8n instance.
```

Comment:
```suggestion
Using community nodes means you are installing unverified code from a public source into your n8n instance. This has some risks.
```
Just a suggestion: I'd remove the first mention of risks to make it more straightforward, and keep the structure cause (unverified code) - effect (risks).